network/src/error.rs

```diff
@@ -50,7 +50,7 @@ pub enum NetworkErrorKind {
     #[fail(display = "Parsing error")]
     ParsingError,
 
-    #[fail(display = "Peer disconnected")]
+    #[fail(display = "Failed to connect to peer")]
```

Comment:
@sherry-x : Why not just say "Peer not connected"? "Failed to connect..." has the connotation that an attempt to connect was made, but was unsuccessful, which is not what is intended from this error type.

Reply:
@bothra90 English is hard lol, I'm happy to update this again if that makes it more clear.

Reply:
Yes, will be great. Thanks!